compiler/rustc_attr/src/builtin.rs

```diff
@@ -48,33 +50,28 @@ pub(crate) enum UnsupportedLiteralReason {
     DeprecatedKvPair,
 }
 
-fn handle_errors(sess: &ParseSess, span: Span, error: AttrError) {
+fn handle_errors(sess: &ParseSess, span: Span, error: AttrError) -> ErrorGuaranteed {
```

Comment:
Should we get rid of this function, and emit the errors directly?

Reply:
Yes, good call. https://github.com/rust-lang/rust/pull/117064